src/app/app.component.spec.ts

```diff
@@ -25,11 +26,4 @@ describe('AppComponent', () => {
     const app = fixture.debugElement.componentInstance;
     expect(app.title).toEqual('myapp');
   });
-
```

Comment:
This test was no longer needed since the app no longer renders "myapp is running" in any way